src/main/java/is/citizen/sdk/util/Constant.java

```diff
@@ -0,0 +1,45 @@
+package is.citizen.sdk.util;
+
+public final class Constant {
+    private Constant() {
+    }
+
+    public static final String  CITIZEN_PRODUCTION_API_HOST               = "api.citizen.is";
+    public static final int     CITIZEN_PRODUCTION_API_PORT               = 443;
+    public static final boolean CITIZEN_PRODUCTION_API_SECURE             = true;
+
+    public static final String  CITIZEN_DEVELOPMENT_API_HOST               = "development.citizen.is";
+    public static final int     CITIZEN_DEVELOPMENT_API_PORT               = 443;
+    public static final boolean CITIZEN_DEVELOPMENT_API_SECURE             = true;
+
+    public static final String  CITIZEN_API_URL                            = "http://localhost:8443/";
```

Comment:
Maybe a better name would be CITIZEN_LOCAL_API_URL ?

Reply:
I removed this - it wasn't needed.